tests/integ/scala/test_dataframe_join_suite.py

```diff
@@ -621,14 +621,14 @@ def test_semi_join_with_columns_from_LHS(
         .select("intcol")
         .collect()
     )
-    assert res == [Row(1), Row(2)]
+    assert sorted(res, key=lambda x: x[0]) == [Row(1), Row(2)]
```

Comment:
Could use `operator.itemgetter(0)` instead?

Reply:
I pushed a commit that switches all appropriate lambdas for [operator.itemgetter](https://docs.python.org/3/library/operator.html#operator.itemgetter)